src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketPluginConfiguration.java

```diff
@@ -40,6 +40,7 @@
     private List<BitbucketServerConfiguration> serverList = new ArrayList<>();
 
     public BitbucketPluginConfiguration() {
+        getConfigFile().getXStream().alias("atl-bbs-configuration", BitbucketPluginConfiguration.class);
```

Comment:
I think this is possibly in the wrong spot?
When I try an upgrade from current master to this one the builds won't run due to
> java.lang.ClassCastException: com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration cannot be cast to com.atlassian.bitbucket.jenkins.internal.config.BitbucketServerConfiguration

Reply:
Oops, fixing…